client/library/library/audits/partydao-2.html

```diff
@@ -0,0 +1,66 @@
+<page
+  clientName="PartyDAO"
+  reportDate="Feb 7, 2022"
+  auditTitle="PartyDAO A-2"
+  auditVersion="1.0.0"
+  repoUrl="https://github.com/PartyDAO/party-protocol"
+  passwordEncrypt="env:PAGE_PASS_PARTYDAO_2"
+  layout="/library/audits/_layout.html"
+  customRepoInfo
+>
+
+<content-for name="schedule">
+  The security audit was performed by the Macro security team from January 16, 2023 to January 27, 2023.
+</content-for>
+
+<content-for name="spec">
+  <ul>
+    <li>Discussions on Telegram with the {{page.clientName}} team.</li>
+    <li>Available documentation in the pull request.</li>
+  </ul>
+</content-for>
+
+<content-for name="repo-info">
+    <ul>
+      <li>
+        <b>Pull Request:</b>
+        <a href="https://github.com/PartyDAO/party-protocol/pull/211" target="_blank">
+          Version 1.1
+        </a>
+      </li>
+      <li class="break-words break-all">
+        <b>Commit Hash:</b>
+        <code>991cf2f4bb9ff785a39a2fb4dd19900beaa2cfbc</code>
+      </li>
+      <li>
+        <b>Details: </b>
+        <ul>
+        <li>The new crowdfund type that can buy multiple NFTs and enter the governance stage with them (CollectionBatchBuyCrowdfund)</li>
+        <li>The new crowdfund that can continue bidding on another auction of the same collection if it doesn't win instead of ending (RollingAuctionCrowdfund)</li>
+        <li>Addition of contributeFor() to crowdfunds to allow contributing on other's behalf</li>
+        <li>Addition of ability for crowdfunds to set min/max contributions per address</li>
+        <li>Allowing host to bid custom amounts in case of auction crowdfunds</li>
+        <li>Support for ERC1155 listing on OpenSea</li>
+        <li>Support for dutch auction listings on OpenSea</li>
+        <li>Removal of useless listPrice param from Fractional proposals</li>
+        </ul>
+      </li>
+    </ul>
+  </content-for>
+
+<content-for name="source-code">  
+  <template type="file-hashes">
```

Comment:
We could write only the name of the contracts instead of the path; the whole path renders quite odd in the final format